src/precompile.js

```diff
@@ -0,0 +1,3 @@
+const jade = require('jade');
+
+module.exports = (template, options) => jade.compileClient(template, options);
```

Comment:
I remember now what the filename is required for! It is used for jade partials. That needs to be the full path of the file and the compiler needs it for where to look at. Handlebars doesn't actually need it as it doesn't allow partials.

So, for achieving feature parity with this: https://github.com/opentable/oc/blob/master/src/cli/domain/package-template.js#L22-L27 we can require the viewPath in the options, but everything else should be here imho, as not really interesting or relevant for OC:

```js
options = {
  filename: viewPath,
  compileDebug: false,
  name: 't'
}
+module.exports = (template, options) => jade.compileClient(template, options).toString().replace('function t(locals) {', 'function(locals){');
```